karmada-division-deploy-example/index.json

```diff
@@ -0,0 +1,36 @@
+{
+    "title": "deploy workload through karmada divided type",
```

Comment:
Same as the other sceanrio. People might not interested in the propagation mode is `divided` or `duplicated`.

Reply:
The title field will be in the Killercodade scene name. If the previous one is called `HA workload deployment`, then this one should be different from the previous one's name